diff --git a/node_modules/@medusajs/admin-ui/ui/src/components/organisms/topbar/index.tsx b/node_modules/@medusajs/admin-ui/ui/src/components/organisms/topbar/index.tsx
index a2a76f6..ef66d79 100644
--- a/node_modules/@medusajs/admin-ui/ui/src/components/organisms/topbar/index.tsx
+++ b/node_modules/@medusajs/admin-ui/ui/src/components/organisms/topbar/index.tsx
@@ -32,20 +32,9 @@ const Topbar: React.FC = () => {
     <div className="min-h-topbar max-h-topbar pr-xlarge pl-base bg-grey-0 border-grey-20 sticky top-0 z-40 flex w-full items-center justify-between border-b">
       <SearchBar />
       <div className="flex items-center">
-        <Button
-          size="small"
-          variant="ghost"
-          className="mr-3 h-8 w-8"
-          onClick={() => setShowSupportForm(!showSupportform)}
-        >
-          <HelpCircleIcon size={24} />
-        </Button>
-
-        <NotificationBell
-          onClick={onNotificationBellClick}
-          variant={"ghost"}
-          hasNotifications={!!batchJobs?.length}
-        />
+
+
+
       </div>
       {showSupportform && (
         <MailDialog
